composer.json

```diff
@@ -32,41 +39,41 @@
         "amenadiel/jpgraph": "^3.6",
         "clue/socket-raw": "^1.4",
         "dapphp/radius": "^2.0",
-        "darkghosthunter/larapoke": "^3.1",
+        "darkghosthunter/larapoke": "^4.1",
         "doctrine/dbal": "^2.9",
         "easybook/geshi": "^1.0.8",
         "ezyang/htmlpurifier": "^4.8",
         "fico7489/laravel-pivot": "^3.0",
+        "fideloper/proxy": "^4.0",
         "influxdb/influxdb-php": "^1.14",
-        "laravel/laravel": "5.8.*",
+        "laravel/framework": "^6.18",
+        "laravel/tinker": "^2.0",
         "martinlindhe/laravel-vue-i18n-generator": "^0.1.42",
         "oriceon/toastr-5-laravel": "dev-master",
         "pear/console_color2": "^0.1",
         "pear/console_table": "^1.3",
         "php-amqplib/php-amqplib": "^2.0",
         "phpmailer/phpmailer": "~6.0",
         "rmccue/requests": "^1.7",
-        "spatie/laravel-cors": "1.3.*",
+        "spatie/laravel-cors": "^1.6",
         "symfony/yaml": "^4.0",
         "tecnickcom/tcpdf": "~6.2.0",
-        "tightenco/ziggy": "^0.7.1",
-        "wpb/string-blade-compiler": "3.8.x-dev",
```

Comment:
string-blade-compiler was removed, re-add it, but looks like upstream has been updated with Laravel 6.0 support.  https://github.com/TerrePorter/StringBladeCompiler/commit/d067645a2601be8256ae758d1612e8fda197cc56

Reply:
When adding it with `FORCE=1 composer require wpb/string-blade-compiler="4.0.x-dev"`
I get;
```
> Illuminate\Foundation\ComposerScripts::postAutoloadDump
> @php artisan package:discover --ansi

   Illuminate\Contracts\Container\BindingResolutionException  : Target class [blade.compiler] does not exist.

  at /Users/jellyfrog/code/librenms/vendor/laravel/framework/src/Illuminate/Container/Container.php:805
    801|
    802|         try {
    803|             $reflector = new ReflectionClass($concrete);
    804|         } catch (ReflectionException $e) {
  > 805|             throw new BindingResolutionException("Target class [$concrete] does not exist.", 0, $e);
    806|         }
    807|
    808|         // If the type is not instantiable, the developer is attempting to resolve
    809|         // an abstract type such as an Interface or Abstract Class and there is

  Exception trace:

  1   Illuminate\Foundation\Application::Illuminate\Foundation\{closure}(Object(DarkGhostHunter\Larapoke\LarapokeServiceProvider))
      [internal]:0

  2   ReflectionException::("Class blade.compiler does not exist")
      /Users/jellyfrog/code/librenms/vendor/laravel/framework/src/Illuminate/Container/Container.php:803

  Please use the argument -v to see more details.
```

Note that It talks about LarapokeServiceProvider, but I'm guessing thats a side effect.

Reply:
This was the culpit, I removed it for now... https://github.com/librenms/librenms/pull/8907.  
@murrant thoughts?

Reply:
It now uses auto-registration.  The manual binding isn't required.  That is the correct fix.